backend/alerts/backend-prometheusRule.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    app.kubernetes.io/name: kube-prometheus
+    app.kubernetes.io/part-of: kube-prometheus
+    prometheus: k8s
+    role: alert-rules
+  name: backend-monitoring-rules
+  namespace: monitoring
+spec:
+  groups:
+  - name: backend
+    rules:
+    - alert: BackendLatency
+      expr: |
+        histogram_quantile(0.95, rate(backend_operations_duration_seconds_bucket[1h])) > 1
+      for: 15m
+      labels:
+        severity: critical
```

Comment:
All alerts and tests should be severity: 'info' so they don't page right now

Reply:
Done!